src/ObjectListView3/ObjectListView.py

```diff
@@ -4113,9 +4113,9 @@ def RepopulateList(self):
         """
         Remember the given model objects so that they can be displayed when the next update cycle occurs
         """
-        if self.freezeUntil < time.clock():
+        if self.freezeUntil < time.process_time_ns():
```

Comment:
Why aren't we using `perf_counter()` here as well?

Reply:
After reading about this more, it sounds like both `perf_counter()` and `process_time()` will work:

https://stackoverflow.com/questions/52222002/what-is-the-difference-between-time-perf-counter-and-time-process-time

I am not sure which is preferable here though

Reply:
That's what I asked in my original https://github.com/driscollis/ObjectListView3/pull/20#issue-2407060692, because I am not sure either. `process_time_ns()` just looked the right to me compared to `perf_counter_ns()`, because with `perf_counter_ns()`, it can time out, if the user is doing some other heavy stuff, but with `process_time_ns()`, it always get the same amount of CPU time, to finish its task.

Reply:
I am okay with this. If we see problems, we know how to fix it anyway